canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -276,13 +276,11 @@ class Toolbar extends React.Component {
 		this.leftBar = this.props.config.leftBar || [];
 		this.rightBar = this.props.config.rightBar || [];
 		this.rightBar = [...this.rightBar].reverse() || [];
-
 		const leftItems = this.generateToolbarItems(this.leftBar, false, true);
 		const rightItems = this.generateToolbarItems(this.rightBar, false, false);
-
 		const canvasToolbar = (
 			<ReactResizeDetector handleWidth onResize={this.onToolbarResize}>
-				<div className="toolbar-div" instanceid={this.props.instanceId}>
+				<div className={`${this.props.size === "sm" ? "toolbar-div toolbar-sm-div" : "toolbar-div"}`} instanceid={this.props.instanceId}>
```

Comment:
We usually do this kind of expression before the JDX is constructed and put the result in a variable, and then refer to the variable in the JSX. I'm not sure why you need the back ticks in here. I think if you move the expression outside the JSX you will definitely not need them.

Reply:
Wrapped the logic into `toolbarClass `
<img width="855" alt="image" src="https://user-images.githubusercontent.com/89609176/181652791-59c17fd2-05b6-463c-a9cc-61f242948754.png">